src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs

```diff
@@ -2,30 +2,21 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Runtime.InteropServices;
-using System.Text;
-using Xunit;
-
 namespace System.Globalization.Tests
 {
     internal static class NumberFormatInfoData
     {
-        private static readonly bool s_isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-        private static readonly int s_WindowsVersion = PlatformDetection.WindowsVersion;
-        private static readonly bool s_isOSX = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
-        private static readonly Version s_OSXKernelVersion = GetOSXKernelVersion();
```

Comment:
I didn't really think that creating a duplicate variable for something that is already cached(/will be when I undo my changes). There's no real change tbh